### DISCUSSION TITLE
Can an agent be both a competitor and a default agent in a simulation?

### DISCUSSION THREAD

**Discussion by rezunli96** [yasserfarouk/scml#26] [Q&A] [answered]
I guess it cannot in oneshot and std track right? Otherwise maybe it makes no difference than col

**Comment by yasserfarouk** [comment, marked as answer]
It cannot in any of the three tracks. As you said it breaks the condition of one agent per simulation for std/oneshot. It is not a good idea in collusion as well because there is no way for your code to know which agents are default agents so it may not be able to collude effectively if money transfers are part of the collusion strategy.
